build-system/pr-check/build-targets.js

```diff
@@ -207,6 +208,9 @@ const targetMatchers = {
       file.startsWith('build-system/test-configs')
     );
   },
+  [Targets.JSON_FILES]: (file) => {
+    return file.endsWith('.json');
```

Comment:
This is a good point. How does `ajv` handle json files for which there is no schema? If the default check is still useful, it may be okay to leave this as a broad matcher. If not, maybe create a list called `jsonCheckPaths` in `build-system/test-configs/config.js` and only check those files?